sites/all/modules/paragraphs/reol_rec_material_paragraph/reol_rec_material_paragraph.features.field_instance.inc

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @file
+ * reol_rec_material_paragraph.features.field_instance.inc
+ */
+
+/**
+ * Implements hook_field_default_field_instances().
+ */
+function reol_rec_material_paragraph_field_default_field_instances() {
+  $field_instances = array();
+
+  // Exported field_instance:
+  // 'paragraphs_item-recommended_material-field_rec_material'.
+  $field_instances['paragraphs_item-recommended_material-field_rec_material'] = array(
+    'bundle' => 'recommended_material',
+    'default_value' => NULL,
+    'deleted' => 0,
+    'description' => 'Kopier URLen på det materiale du vil referere til.',
```

Comment:
Descriptions and labels should be wrapped in t() and translated.

Reply:
Tell that to features.

Field labels are not translatable without the i18n_field module, so we're living with danish strings.